src/Service/ApiClient.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare (strict_types=1);
+
+namespace Service;
+
+
+class ApiClient {
```

Comment:
Zastanawiam się czy klasa typu `Client` powinna mieć metody typu get, post, update, delete. Czy to nie jest logika pod repozytoria? Repozytorium przyjmuje w constructorze clienta który jedyne co robi to doRequest na odpowiedni host z headerami. Dzięki takiemu podejściu można te metody poprzenosić do mniejszych repozytoriów. Temat pod dyskusję.